homeassistant/components/template/config.py

```diff
@@ -163,10 +155,12 @@ async def _async_resolve_blueprints(
         raw_config = dict(config)
 
     if is_blueprint_instance_config(config):
-        config = TEMPLATE_BLUEPRINT_INSTANCE_SCHEMA(config)
         blueprints = async_get_blueprints(hass)
 
         blueprint_inputs = await blueprints.async_inputs_from_config(config)
+        blueprint_inputs = await blueprints.async_inputs_from_config(
+            _backward_compat_schema(config)
```

Comment:
I don't understand how this solves the problem described in the PR description.

Reply:
This is to make sure it's possible to override both using the singular (`trigger`, `condition`) and plural (`triggers`, `conditions`) style configuration.